packages/blocks/src/store/reducer.js

```diff
@@ -34,7 +34,10 @@ export function blockTypes( state = {}, action ) {
 		case 'ADD_BLOCK_TYPES':
 			return {
 				...state,
-				...keyBy( action.blockTypes, 'name' ),
+				...keyBy(
+					map( action.blockTypes, ( blockType ) => omit( blockType, 'styles ' ) ),
```

Comment:
@youknowriad This looks like a typo, should `'styles '` be `'styles'`?

It's present in current releases:

https://github.com/WordPress/gutenberg/blob/71bf1b5146d5dd0fc15792c1a98fc19df25b062f/packages/blocks/src/store/reducer.js#L58

It looks like this was intended to exclude the `styles` property in `blockTypes` state (it's moved to `blockStyles`), but it's present:

```js
wp.data.select('core/blocks').getBlockType("core/image").styles.length
// 2
```

Reply:
definitely looks like a typo yes, do you know what concrete impact this has?

Reply:
I noticed it when inspecting some code, I don't have much idea of the impact it has.

It does appear to duplicate the state stored in `blockStyles`:

```js
wp.data.select('core/blocks').getBlockType("core/image").styles
// [{"name":"default","label":"Default","isDefault":true},{"name":"rounded","label":"Rounded"}]

// Gives same result
wp.data.select('core/blocks').getBlockStyles("core/image")
// [{"name":"default","label":"Default","isDefault":true},{"name":"rounded","label":"Rounded"}]

// But not referential equality
wp.data.select('core/blocks').getBlockType("core/image").styles === wp.data.select('core/blocks').getBlockStyles("core/image")
// false
```